kernel/src/main/java/org/kframework/backend/kore/ModuleToKORE.java

```diff
@@ -577,7 +585,11 @@ private void convertRule(Rule rule, boolean heatCoolEq, Sort topCellSort, Map<St
                     .map(i -> (NonTerminal) i)
                     .map(NonTerminal::sort).collect(Collectors.toList());
             productionLabel = production.klabel().get();
-            if (isFunction(prod) || rule.att().contains(Attribute.ANYWHERE_KEY)) {
+            if (ConstructorChecks.isBuiltinLabel(productionLabel)) {
```

Comment:
Why is this necessary and is it possible we could instead improve the generic handling of the poly attribute?